composer/algorithms/blurpool/blurpool_layers.py

```diff
@@ -27,7 +27,7 @@ def _padding_for_filt_2d_same(filt: torch.Tensor):
         raise IndexError(f"Filter must have odd height; got {h}")
     if w % 2 == 0:
         raise IndexError(f"Filter must have odd width; got {w}")
-    return (h // 2, w // 2)
+    return int(torch.div(h, 2)), int(torch.div(w, 2))
```

Comment:
> UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').

https://ci.int.mosaicml.com/blue/organizations/jenkins/internal%2Fmosaicml%2Fcomposer/detail/PR-1050/10/tests/

Reply:
Looks like PyTorch plans to keep `floordiv`, and then raise an error for negative values before eventually correcting the `floordiv` behavior (https://github.com/pytorch/pytorch/issues/43874). Since the tensor shapes here can never be negative, suggest we keep the original syntax.

Reply:
Sorry, I'm confused -- it seems like the comment here suggests that users should use `torch.div` explicitly? https://github.com/pytorch/pytorch/issues/43874#issuecomment-965222876

> For now we encourage users to follow the deprecation note and replace uses of floor division with a call to div() and the appropriate rounding mode.